client/nimiq-client.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Nimiq's Rust client
+After=syslog.target network.target remote-fs.target nss-lookup.target
+
+[Service]
+Type=forking
+PIDFile=/run/nimiq-client.pid
```

Comment:
I'm not sure if we are creating this file, if not, this should probably be removed.